sbt-plugin/src/main/scala/scala/scalajs/sbtplugin/ScalaJSPluginInternal.scala

```diff
@@ -96,6 +96,29 @@ object ScalaJSPluginInternal {
     incOptions.withNewClassfileManager(newClassfileManager)
   }
 
+  def packageJSDependenciesSetting(taskKey: TaskKey[File], cacheName: String,
+      getLib: ResolvedJSDependency => VirtualJSFile) = {
```

Comment:
Should be private.
Also, would use a result type, even though it's private, given that it's quite big.